### PR TITLE
Tests created for related products endpoint (b2b)

### PR DESCRIPTION
**Pull request by AnastasiiaKSpryker**
New API tests have been added for related products endpoint in b2b env

TBD
none

## Checklist

- [ ]  I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot [hunk 1]
```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
+Product_has_related_products
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot:6]: tests for includes are missing. please add tests with includes for all includes listed in swagger

File: resources/environments/environments_api_b2b.json [hunk 2]
```diff
@@ -125,6 +125,8 @@
         "product_with_relations_related_products_sku": "M29529",
         "product_with_relations_alternative_products_sku": "M21100",
         "product_with_relations_upselling_sku": "M29502",
+        "product_related_to_abstract": "M29501",
```
Comment (AntonShopin) [inline resources/environments/environments_api_b2b.json:128]: please change the name of this variable and the one for the name. We have 2 types of relations: related products and up-sells. 'product_related_product_with_related_relation_sku' sounds better

File: tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot (hunk 1 above)
Comment (AnastasiiaKSpryker) [reply tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot:6]: Test for included was added in last commit

File: tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    And Response body parameter should not be EMPTY:    [data][0][attributes][attributes]
+    And Response body parameter should not be EMPTY:    [data][0][attributes][attributes][brand]
+    And Response should contain the array of a certain size:    [data][0][attributes][superAttributesDefinition]    0
+    And Response should contain the array of a certain size:    [data][0][attributes][superAttributes]    0
+    And Response should contain the array of a certain size:    [data][0][attributes][attributeMap]    4
+    And Response should contain the array of a certain size:    [data][0][attributes][attributeMap][super_attributes]   0
+    And Response should contain the array of a certain size:    [data][0][attributes][attributeMap][product_concrete_ids]   1
+    And Response should contain the array of a certain size:    [data][0][attributes][attributeMap][attribute_variants]   0
+    And Response should contain the array of a certain size:    [data][0][attributes][attributeMap][attribute_variant_map]   0 
+    And Response should contain the array of a certain size:    [data][0][attributes][metaTitle]    0
```
[the remark this answers is not in the document]
Comment (AnastasiiaKSpryker) [reply tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot:29]: Certain size usage was updated to 'larger than a certain size' keyword in order to make tests more generic

File: tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    And Response should contain the array of a certain size:    [data][0][attributes][metaTitle]    0  
+    And Response body parameter should not be EMPTY:    [data][0][attributes][metaKeywords]
+    And Response body parameter should not be EMPTY:    [data][0][attributes][metaDescription]
+    And Response body parameter should not be EMPTY:    [data][0][attributes][attributeNames]
+    And Response body parameter should not be EMPTY:    [data][0][attributes][attributeNames][brand]
+    And Response body parameter should not be EMPTY:    [data][0][attributes][url]     
+    And Each array element of array in response should contain property:    [data]    type
+    And Each array element of array in response should contain property:    [data]    id
+    And Each array element of array in response should contain property:    [data]    attributes   
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    sku
```
[the remark this answers is not in the document]
Comment (AnastasiiaKSpryker) [reply tests/api/b2b/glue/general_product_endpoints/related_products/positive.robot:37]: Added verification that each name and sku in response array are not empty

File: resources/environments/environments_api_b2b.json (hunk 2 above)
Comment (AnastasiiaKSpryker) [reply resources/environments/environments_api_b2b.json]: Variable renamed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**